src/analysis/retail/druid/restoration/modules/spells/VotI2pc.tsx

```diff
@@ -0,0 +1,91 @@
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import { calculateEffectiveHealing } from 'parser/core/EventCalculateLib';
+import SPELLS from 'common/SPELLS';
+import { TIERS } from 'game/TIERS';
+import Events, { HealEvent } from 'parser/core/Events';
+import BoringValueText from 'parser/ui/BoringValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import StatTracker from 'parser/shared/modules/StatTracker';
+import HIT_TYPES from 'game/HIT_TYPES';
+import { formatNumber } from 'common/format';
+import ItemPercentHealingDone from 'parser/ui/ItemPercentHealingDone';
+
+const TWO_PIECE_CRIT_BONUS = 0.08;
+
+const TWO_PIECE_SPELLS = [
+  SPELLS.REJUVENATION,
+  SPELLS.REJUVENATION_GERMINATION,
+  SPELLS.LIFEBLOOM_BLOOM_HEAL,
+  SPELLS.LIFEBLOOM_HOT_HEAL,
+  SPELLS.TRANQUILITY_HEAL,
+  SPELLS.EFFLORESCENCE_HEAL,
+  SPELLS.WILD_GROWTH,
+];
+
+/**
+ **Vault of the Incarnates 2pc**
+ *
+ * Rejuvenation, Lifebloom, Wild Growth, Efflorescence, and Tranquility chance to critically heal is increased by 8%.
+ */
+class VotI2pc extends Analyzer {
+  static dependencies = {
+    stats: StatTracker,
+  };
+  protected stats!: StatTracker;
+  has2Piece: boolean = true;
+  twoPieceHealing: number = 0;
+
+  get critPercentage() {
+    return this.stats.currentCritPercentage;
+  }
+
+  get critIncrease() {
+    return TWO_PIECE_CRIT_BONUS / (TWO_PIECE_CRIT_BONUS + this.critPercentage);
+  }
+
+  get totalHealing() {
+    return this.twoPieceHealing;
+  }
+
+  constructor(options: Options) {
+    super(options);
+    this.has2Piece = this.selectedCombatant.has2PieceByTier(TIERS.T29);
+    this.active = this.has2Piece;
```

Comment:
has2Piece isn't doing anything, it's setting active and that's it.
Try replacing both these lines with `this.active = this.selectedCombatant.has2PieceByTier(TIERS.T29);`